Tests/Snapper_test.py

```diff
@@ -2,6 +2,11 @@
 from Snapper.Snapper import Snapper
 
 
+class TestManager:
+    def on_data(self, snapshot):
+        self.snapshot = snapshot
```

Comment:
The point of this class is to facilitate testing the Snapper's ability to push snapshots by capturing the fact that manager's on_data method was called correctly. At no point in this test file is Manager meant to be tested.

That said, I agree that this needs to be changed in some way. Perhaps use normal manager class, but then MagicMock the on_data method and use assert_called_with instead would be more appropriate?